docs/_articles/en/einstein/einstein-apex.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: Apex Code Generation
+title: Natural Language to Code Generation
 lang: en
 ---
 
-## Generate Apex Code
+## Generate Apex Code from Natural Language Prompts
 
 Use the Einstein for Developers side bar to write a question or an instruction that describes the task for which you'd like to receive an Apex code suggestion and press **Ask**. Copy the code suggestion you received and paste it into an Apex file to use as "starter" code.
```

Comment:
Can we update the screenshot within this section and make it a bit smaller? It has the original issue with the backticks and we've updated the icon recently. Let me know if you need any help grabbing it!

Reply:
Can I get an updated screenshot @AnanyaJha ? Thank you!